src/component/playground/Playground/Playground.tsx

```diff
@@ -103,88 +106,115 @@ export const Playground: VFC<IPlaygroundProps> = () => {
         }
     };
 
+    const onSubmit: FormEventHandler<HTMLFormElement> = async event => {
+        event.preventDefault();
+
+        await evaluatePlaygroundContext(
+            environment,
+            projects,
+            context,
+            setURLParameters
+        );
+    };
+
+    const setURLParameters = () => {
+        searchParams.set('context', encodeURI(context || '')); // always set because of native validation
+        searchParams.set('environment', environment);
+        if (
+            Array.isArray(projects) &&
+            projects.length > 0 &&
+            !(projects.length === 1 && projects[0] === '*')
+        ) {
+            searchParams.set('projects', projects.join(','));
+        } else {
+            searchParams.delete('projects');
+        }
+        setSearchParams(searchParams);
+    };
+
+    const resolveFormWidth = () => {
+        if (results && !matches) {
+            return '35%';
+        }
+
+        return 'auto';
+    };
+
+    const resolveResultsWidth = () => {
+        if (matches) {
+            return '100%';
+        }
+
+        if (results && !matches) {
+            return '65%';
+        }
+
+        return '50%';
+    };
+
+    const resolveLayout = () => {
```

Comment:
I think this can be inlined. `sx={{ display: 'flex', flexDirection: !matches ? 'row' : 'column' }}`

Reply:
Agree